pkgs/top-level/all-packages.nix

```
@@ -31052,6 +31052,8 @@ with pkgs;
 
   mrrescue = callPackage ../games/mrrescue { };
 
+  multimc = callPackage ../games/multimc { };
```

```suggestion
  multimc-bin = callPackage ../games/multimc { };
```

Since this is not being built from source for a reason other than the build from source being too cumbersome or its infrastructure being missing from nixpkgs, this package should clearly hold the `-bin` suffix, just like `firefox-bin` etc.

---

Package has been renamed to `multimc-bin`.